src/markdown.ts

```diff
@@ -297,6 +298,7 @@ export async function parseMarkdown(
     html,
     data: isEmpty(parts.data) ? null : parts.data,
     title: parts.data?.title ?? findTitle(tokens) ?? null,
+    draft: parts.data?.draft,
```

Comment:
It shouldn’t be necessary to expose this here, since it’s already exposed via `page.data`.

```suggestion
```